src/chromo/models/pythia8.py

```diff
@@ -165,6 +165,15 @@ def _set_kinematics(self, kin):
     def _set_stable(self, pdgid, stable):
         self._pythia.particleData.mayDecay(pdgid, not stable)
 
+    def _get_stable(self):
+        r = set()
+        for p in self._pythia.particleData.all():
+            if p.tau0 > 1e-5 and not p.mayDecay:
```

Comment:
Isn't `not p.mayDecay` enought? Or at least p.tau0 > 1e-5 **or** not p.mayDecay. Why `and` is here?

Reply:
I thought like you, but surprisingly, one needs both. If I remember correctly, without the p.tau0 > 1e-5 condition you also get all the naturally stable particles like gamma, p, etc, which should have tau0 == inf, but have tau0 == 0.